api/graphql/reservation_units/reservation_unit_serializers.py

```diff
@@ -258,12 +269,15 @@ class ReservationUnitCreateSerializer(ReservationUnitSerializer, PrimaryKeySeria
         ),
     )
 
-    payment_type = ChoiceCharField(
+    # Using ListField here because MultipleChoiceField is such
+    # a pain in the a** and I couldn't make it work
+    payment_types = serializers.ListField(
```

Comment:
Would it work to do this like: 
```
serializers.ListField(
        child=serializers.PrimaryKeyRelatedField(queryset=ReservationUnitPaymentType.objects.all()),
        source="payment_types",
        .....     
    )
```
Since now the payment type is a model and its primary key is `code`

Reply:
Seems to work! 👏